README.md

```diff
@@ -1,3 +1,92 @@
 # Ice Floe Tracker Pipeline
 
 This repository contains the processing pipeline for IceFloeTracker.jl and ancillary scripts.
+
+## SOIT Integration
+
+The [Satellite Overpass Identification Tool](https://zenodo.org/record/6475619#.ZBhat-zMJUe) is called to generate a list of satellite times for both Aqua and Terra in the area of interest. This program is written in Python and its dependencies are added to the `oscar-env.yaml` that we build in the next section.
+
+**Note:** the `pass_time_cylc.py` script in this project can be adapted to include additional satellites available in the [space-track.org](https://www.space-track.org/) repository.
+
+
+## Cylc to run the pipeline
+
+Cylc is used to encode the entire pipeline from start to finish. Cylc relies on the command line scripts to automate the pipeline. The `flow.cylc` file should be suitable for runs on HPC systems. To run Cylc locally, there are a few commands to run from a terminal in the root directory of this project:
+
+```
+cylc install -n <workflow-name> ./cylc
+cylc graph <workflow-name> #install graphviz locally
+cylc play <workflow-name>
+cylc tui <workflow-name>
+```
+The text-based user interface provides a simple way to watch the status of each task called in the `flow.cylc` workflow. Use arrow keys to investigate each task as see more [here](https://cylc.github.io/cylc-doc/latest/html/7-to-8/major-changes/ui.html#cylc-tui).
+![tui](tui-example.png)
+
+Remember to remove the `images` and `output` folders from the root project directory and use a new workflow name when running the pipeline again.
```

Comment:
I am guessing this is necessary if a workflow has been run already, but if I were a new user I don't think I would not understand this note.

Reply:
Good catch, the folders will overwrite now after the changes to fetchdata, so I deleted this.